Formula/just.rb

```diff
@@ -0,0 +1,19 @@
+class Just < Formula
+  desc "🤖 Just a command runner"
+  homepage "https://github.com/casey/just"
+  url "https://github.com/casey/just.git",
+      :tag => "v0.3.1",
+      :revision => "8fa91a4eb14e576c115a6f1714f27509bd0ed5d9"
+  head "https://github.com/casey/just.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/just"
+  end
+
+  test do
+    system "#{bin}/just", "--version"
```

Comment:
Any chance of a better test here? Looking at the upstream `README` it looks like something more functionality-based could be done?

Reply:
Let's see, I got a hint from the author on this: https://github.com/casey/just/issues/153#issuecomment-335925847